app/design/frontend/base/default/template/oyst/oneclick/cart/redirect.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Could you move `redirect.phtml` in **checkout** folder like `template/oyst/oneclick/checkout/cart/redirect.phtml` and same for `oneclick.phtml` then fix stuff if needed.